proto/rill/runtime/v1/resources.proto

```diff
@@ -499,3 +500,17 @@ message ExecutionError {
 message CharLocation {
   uint32 line = 1;
 }
+
+message ConnectorSpec {
+  string name = 1;
```

Comment:
It doesn't need a `name` field here because every resource already has a name (in `meta.name.name`)